src/sign.ts

```diff
@@ -7,6 +7,11 @@ import { getPlatformInfo, setupEndorctl } from "./utils";
 // Sign options
 function get_sign_options(options: any[]): void {
   const ARTIFACT_NAME = core.getInput("artifact_name");
+  const CERTIFICATE_OIDC_ISSUER = core.getInput("certificate_oidc_issuer");
+  const SOURCE_REPOSITORY_REF = core.getInput("source_repository_ref");
```

Comment:
The inputs `certificate_oidc_issuer` and `source_repository_ref` should be added to `./sign/action.yml`

Reply:
Thank you for the thorough review! I think I've addressed it all.